hangman/src/index.js

```diff
@@ -0,0 +1,288 @@
+import './styles/global.scss';
+import humanHead from './../assets/human-head.svg';
+import humanBody from './../assets/human-body.svg';
+import humanArmLeft from './../assets/human-arm-left.svg';
+import humanArmRight from './../assets/human-arm-right.svg';
+import humanLegLeft from './../assets/human-leg-left.svg';
+import humanLegRight from './../assets/human-leg-right.svg';
+
+// Create main container div
+const mainContainer = document.createElement('div');
+mainContainer.classList.add('main-container');
+
+// Create gallows container
+const gallowsContainer = document.createElement('div');
+gallowsContainer.classList.add('gallows-container');
+gallowsContainer.innerHTML = `
+      <svg id="gallows-svg" width="500" height="581" viewBox="0 0 500 581" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I dislike such inline SVGs very much.

At least you can import your SVG as text and assign it:

```js
import gallowsDrawing from './../assets/my.svg?raw';

gallowsContainer.innerHTML = gallowsDrawing;
```